src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -86,8 +86,8 @@ def config_dashboard(self, daemon_descrs: List[DaemonDescription]) -> None:
     def ok_to_stop(self, daemon_ids: List[str], force: bool = False) -> HandleCommandResult:
         warn, warn_message = self._enough_daemons_to_stop(self.TYPE, daemon_ids, 'Grafana', 1)
         if warn and not force:
-            return HandleCommandResult(-errno.EBUSY, None, warn_message)
-        return HandleCommandResult(0, warn_message, None)
+            return HandleCommandResult(-errno.EBUSY, '', warn_message)
+        return HandleCommandResult(0, warn_message, '')
```

Comment:
shall we use `warn_message` for `stderr` ?

Reply:
I'd rather keep this non-trivial improvement to a separate PR and concentrate on the make check failure.